tensorflow_datasets/structured/tic_tac_toe.py

```diff
@@ -42,8 +69,38 @@ def _info(self):
         builder=self,
         description=_DESCRIPTION,
         features=tfds.features.FeaturesDict({
+            "label": tfds.features.ClassLabel(names=["negative", "positive"]),
+            "features": tfds.features.Sequence(ClassLabel(names=["b", "x", "o"]), length=3*3)
+        }),
+        supervised_keys=("label", "features"),
+        homepage='https://archive.ics.uci.edu/ml/datasets/Tic-Tac-Toe+Endgame',
+        citation=_CITATION,
+    )
+
+  def _split_generators(self, dl_manager):
+    data = dl_manager.download("https://archive.ics.uci.edu/ml/machine-learning-databases/tic-tac-toe/tic-tac-toe.data")
+    return [
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            gen_kwargs={
+
```

Comment:
Great Job on the PR. One minor change would be to remove this unnecessary blank line.